sdk/src/main/java/io/dapr/config/Properties.java

```diff
@@ -24,14 +26,19 @@
   private static final Integer DEFAULT_HTTP_PORT = 3500;
 
   /**
-   * Dapr's default GRPC port.
+   * Dapr's default gRPC port.
    */
   private static final Integer DEFAULT_GRPC_PORT = 50001;
 
   /**
-   * Dapr's default GRPC port.
+   * Dapr's default use of gRPC or HTTP.
+   */
+  private static final DaprApiProtocol DEFAULT_API_PROTOCOL = DaprApiProtocol.GRPC;
+
+  /**
+   * Dapr's default use of gRPC or HTTP for Dapr's method invocation APIs.
    */
-  private static final Boolean DEFAULT_GRPC_ENABLED = true;
+  private static final DaprApiProtocol DEFAULT_API_METHODINVOCATION_PROTOCOL = DaprApiProtocol.HTTP;

```

Comment:
```suggestion
  private static final DaprApiProtocol DEFAULT_API_METHOD_INVOCATION_PROTOCOL = DaprApiProtocol.HTTP;

```

Reply:
I can make this change simply because it is internal and reads as per JAVA convention.

Reply:
I also renamed another one and renamed the env variables.